Tasks/Common/MSBuildHelpers/InvokeFunctions.ps1

```diff
@@ -42,7 +29,14 @@ function Invoke-BuildTools {
                 $splat["LogFileVerbosity"] = $LogFileVerbosity
             }
 
-            Invoke-MSBuild -ProjectFile $file -MSBuildPath $MSBuildLocation -AdditionalArguments $MSBuildArguments -NoTimelineLogger:$NoTimelineLogger @splat
+            if ($Clean) {
+                Invoke-MSBuild -ProjectFile $file -Targets Clean -MSBuildPath $MSBuildLocation -AdditionalArguments $MSBuildArguments -NoTimelineLogger:$NoTimelineLogger @splat
```

Comment:
Doesn't `-Targets Clean` overwrite the default behaviour? With the "Clean" option selected, do we actually still build? My code is no longer being compiled with this option selected so I have deselected it and added "/t:Clean;Build" to MSBuild arguments as a workaround.